### PR TITLE
Setup Graphviz

### PR DESCRIPTION
**Pull request by tawdtangit**
graphviz url
svg to png capability
some refactoring
Change intents to be updated to v14 parameters (Gateway Intents)

### CODE REVIEW

File: graphviz-helper/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  message.channel.send({
+    files: [{
+      attachment: png,
+      name: 'file.png',
+      description: 'A description of the file'
+    }]
+  })
+}
+const createGraph = async (message) => {
+  const dgraph = `
```
Comment (kevvn) [inline graphviz-helper/index.js]: this is just sample data?

File: message-commands/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	// 	}
+	// 	mentionString += ' WAKE UP BITCHES ITS VALOTIME'
+	// 	message.channel.send(mentionString)
+
+	// }
+}
+
+const messageCommandProvider = (message) => {
+    lookForCommand(message)
+    if(message.content=='graph'){
```
Comment (kevvn) [inline message-commands/index.js]: this is so that when someone types the word graph it will trigger this message handler?

Comment (tawdtangit) [reply message-commands/index.js]: yeah but its not permanent
this is just a placeholder

File: graphviz-helper/index.js (hunk 1 above)
Comment (tawdtangit) [reply graphviz-helper/index.js]: yup

**Review by kevvn — APPROVED** [review]

**Pull request merged**